highway/crud_object.go

```diff
@@ -2,22 +2,158 @@ package highway
 
 import (
 	context "context"
+	"errors"
 
+	otv1 "github.com/sonr-io/blockchain/x/object/types"
```

Comment:
rename to 'ot_v1'

Reply:
Should I go ahead and change the name for registery, channels, and buckets to match this scheme?

Reply:
if its low impact we can migrate naming in this PR